file_operations.py

```diff
@@ -0,0 +1,82 @@
+"""Handles file operations, such as naming, downloading and loading"""
+
+
+import json
+import os
+from os.path import join
+from typing import Tuple
+
+from config import IMG_DIR
+
+
+def convert_to_filename(fullTitle: str) -> str:
+    """Converts title to a file name.
+       Used to handle edge cases where the title contains illegal file characters.
+
+    Args:
+        fullTitle (str): The title.
+
+    Returns:
+        str: The file name.
+    """
+    band, song = fullTitle.split(" - ")
+    if band.startswith("DECO"):
+        return "DECO*27" + " - " + song
+    return fullTitle
+
+
+# TODO - Isn't used anywhere atm.
+def convert_from_filename(filename: str) -> str:
+    """Converts filename to a title.
+       Used to handle edge cases where the title contains illegal file characters.
+
+    Args:
+        title (str): The file name.
+
+    Returns:
+        str: The title.
+    """
+    band, song = filename.split(" - ")
+    if band.startswith("DECO"):
+        return "DECO_27" + " - " + song
+    return filename
```

Comment:
arent the naming of these functions reversed? files cant have * in them so conterting to filename should remove the *, and filenames will not have * in them so if we wont to convert from them we have to  add the *.

Reply:
True. Got rid of them